.github/actions/spelling/expect.txt

```diff
@@ -25,6 +26,7 @@ abcdefghijklmnopqrstuvwxyz
 ble
 blog
 btn
+Bubl
```

Comment:
Not "Bubl" but "Bublé".

Reply:
Yep, intentional though, the spell checker doesn't care about `é`

Reply:
OK then!